test/initialization/netsuite_scenarios.py

```diff
@@ -0,0 +1,112 @@
+import pyautogui
+from common import *
+from selenium.webdriver import ActionChains
+from selenium.webdriver.common.by import By
+
+import driver
+from step_definitions import common_steps, script_steps, call_interaction_steps, sf_login_steps
+from step_definitions.adapters import adapter_login_steps, adapter_steps, adapter_worksheet_steps
+from test.initialization import base_setup
+
+
+def check_netsuite_basic_calls():
+    base_setup.set_base_pages(instances=2)
+    common_steps.STARTED_PAGES.append(adapter_login_steps.ADAPTER_LOGIN_PAGE)
+    common_steps.STARTED_PAGES.append(script_steps.SCRIPT_PAGE)
+    common_steps.STARTED_PAGES.append(adapter_steps.ADAPTER_PAGE)
+    common_steps.STARTED_PAGES.append(adapter_worksheet_steps.ADAPTER_WORKSHEET_PAGE)
+    common_steps.STARTED_PAGES.append(call_interaction_steps.CALL_INTERACTION_PAGE)
+    common_steps.STARTED_PAGES.append(sf_login_steps.LOGIN_PAGE)
+
+    lab_config = f"configuration/lab/{get_config_file_section('config.yml', 'configuration').get('lab')}.yml"
+    connector_url_ = get_config_file_section(lab_config, 'configuration').get('connector_url')
+    if connector_url_:
+        call_interaction_steps.CALL_INTERACTION_PAGE.connector_url = connector_url_
+
+    try:
+        sf_login_steps.AGENT_CREDENTIALS = get_config_file_section(lab_config, 'ns_credentials')
+    except KeyError as e:
+        print(f'could not find "credentials" section in {driver.CONFIG_FILE}.\n{e}\nusing the default...')
+        sf_login_steps.AGENT_CREDENTIALS = get_config_file_section('config.yml', 'credentials')
+
+    # find the extension
+    for _driver in driver.DRIVERS:
+        driver_ = driver.DRIVERS.get(_driver).get('instance')
+        set_adapter_shortcut(driver_, "Five9Lab Plus Adapter for NetSuite")
+        set_adapter_url(driver_, set_adapter_shortcut(driver_, "Five9Lab Plus Adapter for NetSuite", True))
+        break
+
+# ----- SETUP ----- #
+def set_adapter_shortcut(driver_, extension_text, return_id=False):
```

Comment:
as you are using the same code, you can call this method direct from adt_plus_scenarios.py initialization module.
or we can create some common utils for adapter initialization.

Reply:
Please check new commit